### ISSUE TITLE
Binary files + git = :-(

### ISSUE THREAD

**Issue by tscanlin** [optimizely/oui-icons#17]
@tomgenoni, @danoc, @daverau-optimizely, @kwalker3690 and @zleach 

I propose we remove the .ai files from this repo and simply rely on the .svg files already there.

Right now it is extremely painful to work with the .ai files in this repo. If anyone else makes changes to the ai files you will have merge conflicts and those conflicts cannot be resolved easily.

I think it would reduce a lot of complexity to just use svg files in here and have the designers save in that format directly since illustrator works fine with .svg's I don't see the issue with this.

The biggest thing I think we lose by removing the .ai file is the ability to see all the icons at once. But I propose we just make an html page where they can all be viewed.

Let me know if I'm missing any other benefits of keeping the .ai files or if you have strong opinions about this.

Thanks!

**Comment by danoc** [comment]
I totally agree! Feel free to make the change of the designers are cool with it and no one else objects.

The AI files could be stored in Dropbox IMO.

**Comment by daverau-optimizely** [comment]
+1 for Dropbox, it's designer-friendly and keeps our repo files cleaner.

**Comment by zleach** [comment]
@tscanlin @danoc @daverau-optimizely Tim will check to see if we can move the .ai to svg, and not loose any of the nice Illustrator features (like artboards). If we can't use SVGs as the line version files, We should move it all to Dropbox, lets just make sure there's a link in the readme.md

**Comment by tscanlin** [comment]
Cool, thank you both for the dropbox suggestion!

I talked this over with Zach (in person) and he seemed cool with the dropbox method as well. I'll plan to switch to that after merging in both open PRs, updating the README with new instructions, and then removing the ai files and moving to a dropbox location (TBD).

**Comment by tscanlin** [comment]
oh, and will look into the single svg file way too, not sure if that will work though.

**Comment by tscanlin** [comment]
I tried saving the .ai file as a svg but when I did all the artboard data was lost (which is needed to export the individual files).

So I made the necessary updates from the open PRs on this repo and put those files in dropbox, here:
https://www.dropbox.com/work/Design/oui-icons/source

Next week I will work with Zach to finalize the folder locations and export all the icons. Then I will update this repo with those icons and remove the .ai files from this repo. And I will update the readme with the new process.
